### PR TITLE
Removes trailing periods from settings labels

### LINKED ISSUE #6220: Settings: Remove periods from the end of all setting labels

### PR DESCRIPTION
**Pull request by MichaelArestad**
Fixes #6220

#### Changes proposed in this Pull Request:
* Removes all trailing periods from settings labels

#### Testing instructions:
* Go to all settings sections and look for any trailing periods on toggle/checkbox labels.

#### Example before:
image

#### Example after:
image

### CODE REVIEW

File: modules/module-headings.php [hunk 1]
```diff
@@ -36,7 +36,7 @@ function jetpack_get_module_i18n( $key ) {
 
 			'custom-content-types' => array(
 				'name' => _x( 'Custom Content Types', 'Module Name', 'jetpack' ),
-				'description' => _x( 'Organize and display different types of content on your site.', 'Module Description', 'jetpack' ),
+				'description' => _x( 'Display different types of content on your site with custom content types.', 'Module Description', 'jetpack' ),
```
Comment (MichaelArestad) [inline modules/module-headings.php]: Not sure why this changed. Ideas?

**Comment by MichaelArestad** [comment]
Looks like a few lines in `module-headings.php` changed. No idea why since it's generated. Ideas?

**Comment by eliorivero** [comment]
If some lines in `module-headings.php` changed then the description in the related module file header changed and that must be probably me, since I updated some of the descriptions to match the texts in the mockups in p6TEKc-HQ-jpopdesignp2.

**Comment by eliorivero** [comment]
Needs rebasing and restoring the `module-headings.php` file.

**Comment by eliorivero** [comment]
Rebased and restored.

**Comment by MichaelArestad** [comment]
@eliorivero You're the best. Thank you.

**Pull request merged**